go_modules/lib/dependabot/go_modules/update_checker.rb

```diff
@@ -4,7 +4,6 @@
 require "dependabot/update_checkers/base"
 require "dependabot/shared_helpers"
 require "dependabot/errors"
-require "dependabot/go_modules/native_helpers"
```

Comment:
I suspect this is an unused import... probably never cleaned up when we previously thinned out other commands from the native helpers...